### PR TITLE
Add support for LDAPS (SSL/TLS)

### PR DESCRIPTION
**Pull request by vmario89**
Just a feature, requested by a user in 2021 (and by myself in 2023)

### CODE REVIEW

File: docs-web/src/main/webapp/package-lock.json
Comment (jendib) [inline docs-web/src/main/webapp/package-lock.json:1]: Why did you recommit this file? It should not be needed.

File: docs-web/src/main/webapp/src/locale/en.json [hunk 1]
```diff
@@ -286,6 +286,7 @@
       "enabled": "Enable LDAP authentication",
       "host": "LDAP hostname",
       "port": "LDAP port (389 by default)",
+      "usessl": "Aktiviere SSL (ldaps)",
```
Comment (jendib) [inline docs-web/src/main/webapp/src/locale/en.json:289]: it's the english translation file => "Enable SSL (ldaps)"

**Review by jendib — CHANGES_REQUESTED** [review]
Thanks, I left a few comments if you can check it out.

File: docs-web/src/main/webapp/package-lock.json
Comment (vmario89) [reply docs-web/src/main/webapp/package-lock.json:1]: oops. not sure. i guess it is not in my local .gitignore

File: docs-web/src/main/webapp/src/locale/en.json (hunk 1 above)
Comment (vmario89) [reply docs-web/src/main/webapp/src/locale/en.json]: fixed. overseen it. thanks!

File: docs-web/src/main/webapp/package-lock.json
Comment (vmario89) [reply docs-web/src/main/webapp/package-lock.json:1]: i removed it in my latest commit

File: docs-web/src/main/webapp/src/locale/de.json [hunk 2]
```diff
@@ -278,8 +278,23 @@
     "menu_vocabularies": "Vokabulareinträge",
     "menu_configuration": "Einstellungen",
     "menu_inbox": "Posteingang durchsuchen",
+	"menu_ldap": "LDAP Authentifizierung",
```
Comment (jendib) [inline docs-web/src/main/webapp/src/locale/de.json:281]: Incorrect indentation, it should be align with the line above

File: docs-web/src/main/webapp/package-lock.json
Comment (jendib) [inline docs-web/src/main/webapp/package-lock.json:1]: You deleted the file. Instead you should remove the change from your branch

**Review by jendib — CHANGES_REQUESTED** [review]

File: docs-web/src/main/webapp/package-lock.json
Comment (vmario89) [reply docs-web/src/main/webapp/package-lock.json:1]: i am not a git pro. have no idea how to do this :(

**Pull request closed without merging**